app/src/main/java/com/example/mycleaninglog/MainActivity.kt

```diff
@@ -43,7 +42,7 @@ class MainActivity : ComponentActivity() {
                 viewModel.user = user
                 viewModel.listenToMyRooms()
             }
-            val user by viewModel.users.observeAsState(initial = emptyList())
+            val user by viewModel.userList.observeAsState(initial = emptyList())
```

Comment:
users is adequate.  It indicates there are multiple of them.  No need to add the collection type to the variable name as well.